pom.xml

```diff
@@ -7,17 +7,20 @@
     <packaging>pom</packaging>
 
     <properties>
-    	<maven.compiler.release>11</maven.compiler.release>
-        <tycho.version>2.2.0</tycho.version>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<maven.compiler.source>17</maven.compiler.source>
+		<maven.compiler.target>17</maven.compiler.target>
+		<maven.compiler.release>17</maven.compiler.release>
+        <tycho.version>4.0.6</tycho.version>
         <eclipse.cbi>1.3.2</eclipse.cbi>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <p2.repo>http://download.eclipse.org/releases/2022-03</p2.repo>
-        <lyo.repo>file:org.eclipse.lyo.oslc4j.plugins/target/repository</lyo.repo>
```

Comment:
What was using this variable? I think we may have done the build in two stages earlier and the 2nd stage relied on a locally built repo. If it builds and packages now, should be safe to remove.

Reply:
The first stage works fine. In the second stage, there is an error that it could not find the local repo (or it did not recognise a p2 repo on that local path.
Note though that we are not removing <lyo.repo>. I am just replacing the local repo (file) with the real path under http://../edge.
So, we really don't need the 2 stages anymore, if we think it is good enough to not rely on local build first

Reply:
The real problem is that I could not figure why it (tycho?) could not identify the local path. I just could not be bothered.

Reply:
The risk I see with this is that a build of a given commit will be pulling some stuff from edge for the previous commit. Could lead to trouble in the future, esp. with bigger changes.

Reply:
Remember that the "plugins" plugin does not have any code/logic. It is bundling the Lyo SDK.
So, the real risk is that we are pulling from the snapshot versions of the SDK. 
I might be missing something in my logic, but I wonder if it really makes a difference.

Reply:
And I also noticed that we are actually using Lyo 5.1.1.Final. Because the relevant parts for LyoDesigner are quite stable, so there is no reason to even use Snapshot.
https://github.com/eclipse/lyo.designer/pull/279/files#diff-3842ff5d52b15542f0efd42214f37b13cac3054216f29be0c9133029eb897370R26